lib/components/narrative/narrative-itineraries.js

```diff
@@ -30,48 +33,174 @@ class NarrativeItineraries extends Component {
     itineraryClass: DefaultItinerary
   }
 
+  state = {}
+
+  _toggleDetailedItinerary = () => {
+    this.setState({showDetails: !this.state.showDetails})
+  }
+
+  _saveTrip = () => {
+    window.confirm('Are you sure you want to save this trip?')
+  }
```

Comment:
Doesn't save trip, just displays confirmation window.

Reply:
This was built before @binh-dam-ibigroup's work existed. I think we should coordinate on syncing this work outside of this PR.